docs/modules/ROOT/pages/management-center-ldap.adoc

```diff
@@ -41,6 +44,9 @@ The following are the required fields to configure the LDAP Security provider.
 |`groupSearchFilter`
 | LDAP search filter expression to search for the groups.
 
+|`nestedGroupSearch`
```

Comment:
This one is not required, as mentioned in the section above, that's why I didn't document it here. All the fields with the optional ones are included in the Refrence API doc page 😃

Reply:
Hmm I didn't know that, I'll remove it back. How about URL field? It was missing or it's also intentionally removed

Reply:
URL is required, so it was just missing